corehq/apps/export/utils.py

```diff
@@ -57,10 +57,21 @@ def get_default_export_settings_if_available(domain):
     """
     Only creates settings if the domain has the DEFAULT_EXPORT_SETTINGS privilege
     """
+    from corehq.apps.accounting.models import Subscription
     settings = None
     current_subscription = Subscription.get_active_subscription_by_domain(domain)
     if current_subscription and domain_has_privilege(domain, DEFAULT_EXPORT_SETTINGS):
         from corehq.apps.export.models import DefaultExportSettings
         settings = DefaultExportSettings.objects.get_or_create(account=current_subscription.account)[0]
 
     return settings
+
+
+def get_transform_function(func_name):
+    module = importlib.import_module('corehq.apps.export.transforms')
```

Comment:
Why is `importlib` necessary here? Does a normal import not work?
```suggestion
    import corehq.apps.export.transforms as module
```